src/qutip_tensornetwork/core/data/tensor_train/tensor_train.py

```diff
@@ -0,0 +1,289 @@
+from qutip_tensornetwork.core.data.network import Network, _match_edges_by_split
+import tensornetwork as tn
+
+__all__ = ["FiniteTT", "network_to_tt"]
+
+
+class FiniteTT(Network):
+    """Represent finite tensor-trains. This can be either MPS or MPO. A
+    schematic representation of such tensor trains are:
+        |   |   |
+        * - * - *
+        |   |   |
+    where "-" represents an edge and "*" a node. The above graph represents an
+    MPO but MPS can also be represented with this class. The dangling edges
+    represent the ``in_edges`` and ``out_edges`` (which correspond to rows and
+    columns of a matrix when the network is contracted into a 2D array). The
+    bonds connecting the nodes can be accessed with ``bond_edges``. Nodes can
+    also be accessed in a shorted list format using ``node_list``.
+
+    Notes
+    -----
+    Nodes in this class have been named from 0 to L, with L the number of
+    nodes,  as they can be sorted (from left to right). The axis of each node
+    have also been named for easy access. These are named as: {"in", "out",
+    "rbond", "lbond"}.
```

Comment:
I was at first a little bit confused on whether call I should call it "edges" or "axis". I decided for "axis" since "edges" can also have a name but are not named in the tt class. Only the axes have been named. This means that the following can happen: `node_0['rbond'] == node_1['lbond]`. I interpreted this last equality as "the edge of the axis 'rbond' for node_0 is the same as the edge for the axis 'lbond'". Hence, I would prefer to leave it as "axis".

Reply:
Ah, I see "axis" here means "the link between an node and an edge". I'm happy to change it to "axes".